crates/starknet-server/tests/common/mod.rs

```diff
@@ -145,8 +145,8 @@ pub mod util {
 
             let process = Command::new("cargo")
                 .arg("run")
-                .arg("--release")
                 .arg("--")
```

Comment:
using this order of arguments an error arises: "error: unexpected argument '--release' found"

Reply:
My bad screwed up with [revert](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/126/commits/a93bd4078161ade5683e5cf1581cedced25972e7)